src/index.html

```diff
@@ -16,17 +16,28 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
```

Comment:
The class name `body` doesn't represent the meaning of the content. Consider using a more descriptive class name that represents the role or purpose of the element in the context of the interface.